src/aux-server/aux-web/shared/interaction/ClickOperation/BaseFileClickOperation.ts

```diff
@@ -99,6 +100,18 @@ export abstract class BaseFileClickOperation implements IOperation {
             : this.game.getInput().getMouseButtonHeld(0);
 
         if (buttonHeld) {
+            if (
+                this._startFilePos === null &&
+                this._file3D != null &&
+                this._file3D.display != null
+            ) {
+                let tempPos = this._file3D.display.position.clone();
+                this._startFilePos = new Vector2(
+                    Math.round(tempPos.x / 0.4),
+                    Math.round(tempPos.z / -0.4)
+                );
```

Comment:
@shaneYeti The _startFilePos should be calculated from the file's context position in its tags, not the position that the 3D object has. The problem with doing it this way is that you have to reverse the grid scale calculation which is based on a tag on the context file. Grabbing the file position directly is not only easier but also safer.

See: https://github.com/casual-simulation/aux/blob/f6e7b4d0ae55a4820a56852de704ea8c4a2b9c69/src/aux-common/Files/FileCalculations.ts#L1482-L1492

Reply:
Set this up to work with the file's context position